app_server/views/layout.pug

```diff
@@ -9,6 +9,7 @@ head
     link(href="https://fonts.googleapis.com/css?family=Roboto:300,400", rel="stylesheet")
     link(rel='stylesheet', href='/bootstrap/css/bootstrap.min.css')
     link(rel='stylesheet', href='/stylesheets/style.css')
+    script(src='https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js')
```

Comment:
Hm im guessing you need jQuery but its already included at the bottom. I guess we could make a new issue to move all the script files into the `head`

Reply:
You're right I'll remove it